### PR TITLE
deploy prometheus and grafana using tf

### PR DESCRIPTION
**Pull request by AndyBoWu**
This PR is huge, let me know if you need further info.

### CODE REVIEW

File: grafana-deployment/terraform.tfstate.backup [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (LeoHChen) [inline grafana-deployment/terraform.tfstate.backup]: why check-in the backup file?

File: grafana-deployment/terraform.tfstate [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            "user_data_base64": null,
+            "volume_tags": {
+              "Name": "HarmonyNode-monitoring-Volume",
+              "Project": "Harmony"
+            },
+            "vpc_security_group_ids": [
+              "sg-04bd70e941e53e557"
+            ]
+          },
+          "private": "eyJlMmJmYjczMC1lY2FhLTExZTYtOGY4OC0zNDM2M2JjN2M0YzAiOnsiY3JlYXRlIjo2MDAwMDAwMDAwMDAsImRlbGV0ZSI6MTIwMDAwMDAwMDAwMCwidXBkYXRlIjo2MDAwMDAwMDAwMDB9LCJzY2hlbWFfdmVyc2lvbiI6IjEifQ==",
```
Comment (LeoHChen) [inline grafana-deployment/terraform.tfstate]: you sure you want to check in this tfstate file?  what's this private?

File: grafana-deployment/terraform.tfstate.backup (hunk 1 above)
Comment (AndyBoWu) [reply grafana-deployment/terraform.tfstate.backup]: Should not. Adding it to .gitignore.

File: grafana-deployment/terraform.tfstate (hunk 2 above)
Comment (AndyBoWu) [reply grafana-deployment/terraform.tfstate]: .tfstate file is used to keep tracking the instance status on the cloud. Adding it to .gitignore too.

**Comment by AndyBoWu** [comment]
Made some changes to start Grafana at boot. Closing off this PR for now.

**Pull request closed without merging**